src/olympia/devhub/templates/devhub/addons/submit/done.html

```diff
@@ -7,17 +7,23 @@
 {% block primary %}
 {% set version_edit_url = url('devhub.versions.edit', addon.slug, uploaded_version.id) %}
 {% if uploaded_version.channel == amo.RELEASE_CHANNEL_LISTED %}
-  <h3>{{ _("Version Submitted for Review") }}</h3>
-  <p>
-    {{ _("You’re done! This version has been submitted for review. You will be "
-         "notified when the review has been completed, or if our reviewers have "
-         "any questions about your submission.") }}
-  </p>
-  {% if submit_page == 'addon' %}
-  <p>
-    {{ _("Your listing will be more successful by adding a detailed description "
-         "and screenshots. Get your listing ready for publication:") }}
-  </p>
+  {% if submit_page == 'file' %}
+    <p>
+      {{ _("You’re done!") }}
+    </p>
+  {% else %}
+    <h3>{{ _("Version Submitted for Review") }}</h3>
```

Comment:
shouldn't "Submitted" be lower case? I know this is just copy & paste but now that we're touching it...